python/python/ert_gui/shell/simulations.py

```diff
@@ -33,9 +33,18 @@ def ensembleExperiment(self, line):
         simulation_runner = EnkfSimulationRunner(self.ert())
 
         now = time.time()
-
         print("Ensemble Experiment started at: %s" % datetime.now().isoformat(sep=" "))
 
+        print("Create the run path!")
+        iteration_count = self.ert().analysisConfig().getAnalysisIterConfig().getNumIterations()
+        active_realization_mask = BoolVector.createActiveMask("0-%d" % self.ert().getEnsembleSize())
```

Comment:
This should be `"0-%d" % self.ert().getEnsembleSize() - 1` - but even better:
```Python
active_realization_mask = BoolVector( default_value = True , initial_size = self.ert().getEnsembleSize())
```
not any reason to go through the string based construction.